cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -175,4 +208,78 @@ public void resetTxManagerState(HttpRequest request, HttpResponder responder) {
     txClient.resetState();
     responder.sendStatus(HttpResponseStatus.OK);
   }
+
+  @Path("/transactions/pruneinfo")
+  @POST
```

Comment:
This can be a GET call since it just returns information, and we don't change any state, right?